### DISCUSSION TITLE
Question about grass phenology

### DISCUSSION THREAD

**Discussion by hannah0304** [NGEET/fates#1111] [Q&A] [answered]
Hello all,

I am currently conducting point-simulations using FATES across various sites where C3 grass is the dominant vegetation.
I noticed that FATES currently categorizes C3 grass phenology as 'stress-deciduous', responding to both temperature and moisture. I'm curious: should this setting remain constant always?

In my specific sites, C3 grass seems to respond only to temperature, suggesting a seasonal deciduous (or cold-deciduous) behavior in FATES I believe. I have adjusted the binary flag for leaf habit in the parameter file (specifically, 'fates_phen_season_decid' and 'fates_phen_stress_decid') and found no technical issues.

However, I would like to confirm if such modifications are in line with the intended model settings. Could anyone provide some insights?

Thank you.

**Comment by XiulinGao** [comment, marked as answer]
Hi Hannah, 

 Default phenology of C3 grasses in FATES is stress-deciduous (or drought deciduous), which to my knowledge should only respond to change in soil water content or water potential averaged across root profile, depending on how you parameterize it (determined by fates_phen_drought_threshold and fates_phen_moist_threshold in the recent version of FATES; in older version it's only determined by fates_phen_drought_threshold and only meant to be volumetric soil water content). In our California annual grassland simulations, where we also used C3, drought deciduous grasses, seasonal variation in matter and energy fluxes are strongly influenced by the fates_phen_drought_threshold parameters, but you'll also probably have to tune your drought mortality and other drought response parameters (e.g smpsc, smpso, fates_mort_scalar_hydrfailure) in order to see a quick drop in GPP and LAI once dry season starts. Otherwise, there always will be some plants stay active even during dry season, which can be due to combinations of new recruitments and plants that can resist low water availability (depending on how you set your phenology threshold and drought tolerance). 

If your C3 grasses are indeed cold-deciduous grasses instead of drought deciduous, yes, you should change the leaf phenology flag to seasonal deciduous. And for cold deciduous, there are many parameters you can tune to make your case works. those are: phen_chilltemp, phen_coldtemp, phen_mindayson, phen_mindaysoff, phen_ncolddayslim, and maybe cold tolerance and mortality parameters. By changing the leaf phenology flag parameter alone, I doubt the default settings of all the parameters I just mentioned will work perfectly for your case. Hope this can help.

**Reply by hannah0304** [reply]
This is fantastic, thank you so much for your assistance.

Your suggestions are very helpful. As you pointed out, I've been exploring multiple parameter combinations to align the model results with observations. Currently, I'm working with an older version of FATES where drought-deciduous phenology is solely determined by 'fates_phen_drought_threshold'.

Yet, according to the CLM documentation, 'Section 20.4. Stress-DeciduousPhenology',
_"The stress-deciduous phenology algorithm was specifically developed for CLM, partially based on the grass phenology model proposed by White et al. (1997). This algorithm manages phenology for vegetation types such as grasses and tropical drought-deciduous trees that react to both cold and drought stress signals and can experience multiple growing seasons in a year ... For climates that remain warm throughout the year, onset is triggered by soil water availability ... In colder climates, the onset depends on both the accumulated soil temperature summation and adequate soil moisture."_

Upon reviewing the EDPhysiology.F90 code and considering your response, it appears that in FATES, drought-deciduous phenology is determined exclusively by soil moisture. The FATES documentation I have, updated in July this year, mentions "1.13.2 Drought-deciduous Phenology: TBD." While the code can definitely explain the mechanism, perhaps an update in the documentation would be beneficial too?

Thank you again.

**Comment by XiulinGao** [comment]
also, try changing this ED_val_phen_mindayson to phen_doff_time in this line so your phen_midfaysoff parameter can kick into effect when you change it. https://github.com/NGEET/fates/blob/1d18bb69ee47a14467c172879ea0083dc8178012/biogeochem/EDPhysiologyMod.F90#L1112
